build/tasks/RepoTasks.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
This is how other RepoTasks.csproj are setup. Helps us keep task dependencies consistent with what ships in KoreBuild.

```xml
<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$(RepoTasksSdkPath)\Sdk.props" Condition="'$(RepoTasksSdkPath)' != '' "/>

   <PropertyGroup>
       <!-- still required to keep VS happy -->
       <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>

  <Import Project="$(RepoTasksSdkPath)\Sdk.targets" Condition="'$(RepoTasksSdkPath)' != '' "/>
</Project>
```